OpenRA.Game/Sound/Sound.cs

```diff
@@ -101,7 +101,7 @@ public static void SetListenerPosition(WPos position)
 			soundEngine.SetListenerPosition(position);
 		}
 
-		static ISound Play(Player player, string name, bool headRelative, WPos pos, float volumeModifier)
+		static ISound Play(Player player, string name, bool headRelative, WPos pos, float volumeModifier = 1, bool loop = false)
```

Comment:
I'm not sure because I can't think of any other cases to check for default float param values, but this may need to be 1f (again, consistency, not changing the impl).